### DISCUSSION TITLE
Some of crazy ideas: waterfall, scanner.

### DISCUSSION THREAD

**Discussion by pavelkolodin** [AlexandreRouma/SDRPlusPlus#1047] [Ideas]
1. Special waterwall mode: stop waterwall on squelch. Seems simple to implement. Why: would be interesting to see a "spectral picture" for a long period of time when something happens very rarely. Also, waterwall must draw timestamp on each squelch activation.
2. Remember scanner settings. Each start SDR++ gives you 88-108 MHz in scanner settings.
3. Scan lists. In this mode, scanner dont scan, but jumps by records in list.
4. Remember display settings; currently, I start my SDR++ session with setting 4096 FFT buckets (default is 65536).

**Comment by AlexandreRouma** [comment]
1) I don't think anyone would actually use this. You're literally hiding data that could show you there's another signal somewhere else. And if you only have a single signal you care about, why stop the waterfall?
2) The settings aren't remembered on purpose because the scanner is still work in progress. I don't want to deal with setting format conversion when stuff eventually change.
3) Already planned
4) Those settings are already remembered. Check that SDR++ can actually write to the directory where you installed it. If it can't save its config file, it can't remember anything.
